torchaudio/models/_wavernn.py

```diff
@@ -5,101 +5,83 @@
 
 
 class _ResBlock(nn.Module):
-    r"""This is a ResNet block layer. This layer is based on the paper "Deep Residual Learning
-    for Image Recognition". Kaiming He,  Xiangyu Zhang, Shaoqing Ren, Jian Sun. CVPR, 2016.
-    It is a block used in WaveRNN. WaveRNN is based on the paper "Efficient Neural Audio Synthesis".
-    Nal Kalchbrenner, Erich Elsen, Karen Simonyan, Seb Noury, Norman Casagrande, Edward Lockhart,
-    Florian Stimberg, Aaron van den Oord, Sander Dieleman, Koray Kavukcuoglu. arXiv:1802.08435, 2018.
+    r"""ResNet block based on "Deep Residual Learning for Image Recognition"
+
+    The paper link is https://arxiv.org/pdf/1512.03385.pdf. The input signal is a spectrogram.
```

Comment:
nit: after re-reading this, I would simply remove "The input signal is a spectrogram." since you are already describing the arguments below.

Reply:
I agree. I removed this sentence. Thanks.